docs/rfc/active/business_glossary/README.md

```diff
@@ -0,0 +1,358 @@
+```java
+/**
+ * A metadata snapshot for a specific BusinessTerm entity.
+ */
+record BusinessTermSnapshot {
+
+  /**
+   * URN for the entity the metadata snapshot is associated with.
+   */
+  urn: BusinessTermUrn
+
+  /**
+   * The list of metadata aspects associated with the dataset. Depending on the use case, this can either be all, or a selection, of supported aspects.
+   */
+  aspects: array[BusinessTermAspect]
+}
+```
+
+Path : metadata-models/src/main/pegasus/com/linkedin/metadata/aspect/
+
+### BusinessTermAspect
+There will be new aspect defined to capture the required attributes & ownership information
+
+```
+/**
+ * A union of all supported metadata aspects for a BusinessTerm
+ */
+typeref BusinessTermAspect = union[
+  BusinessTermProperties,
+  Ownership
+]
+```
+
+Business Term Entity Definition
+```java
+/**
+ * Data model for a Business Term entity
+ */
+record BusinessTermEntity includes BaseEntity {
+
+  /**
+   * Urn for the dataset
+   */
+  urn: BusinessTermUrn
+
+  /**
+   * Business Term native name e.g. CashInstrument
+   */
+  name: optional string
+
+}
+```
+
+### Entity BusinessTermInfo
+
+```java
+/**
+ * Properties associated with a BusinessTerm
+ */
+record BusinessTermInfo {
+
+  /**
+   * Definition of business term
+   */
+  definition: string
+
+  /**
+   * Source of the Business Term (INTERNAL or EXTERNAL) with default value as INTERNAL
+   */
+  termSource: EnumType
+
+  /**
+   * External Reference to the business-term (URL)
+   */
+  sourceRef: optional string
+
+ /**
+   * URI of the external source when the term is borrowed from external
+   * e.g: fibo-fbc-fi-fi:CashInstrument
+   */
+  sourceURI: optional string
+
+  /**
+   * The abstracted URI such as https://spec.edmcouncil.org/fibo/ontology/FBC/FinancialInstruments/FinancialInstruments/CashInstrument.
+   */
+  uri: optional Uri
+
+}
+
+```
+
+### Business Term Realationship with Owner
+Business Terms will be owened by certain business users
+
+```
+/**
+ * A generic model for the Owned-By relationship
+ */
+@pairings = [ {
+  "destination" : "com.linkedin.common.urn.CorpuserUrn",
+  "source" : "com.linkedin.common.urn.BusinessTermUrn"
+}, {
+   "destination" : "com.linkedin.common.urn.BusinessTermUrn",
+   "source" : "com.linkedin.common.urn.CorpuserUrn"
+} ]
+record OwnedBy includes BaseRelationship {
+
+  /**
+   * The type of the ownership
+   */
+  type: OwnershipType
+}
+```
+
+### Dataset Field related Changes/enhancements
+
+Defining new ```DatasetFiled``` Entity
+
+```java
+record DatasetFieldEntity includes BaseEntity {
+
+  /**
+   * Urn for the dataset
+   */
+  urn: DatasetFieldUrn
+
+  /**
+   * Dataset Field native name e.g. {db}.{table}, /dir/subdir/{name}, or {name}
+   */
+  name: optional string
+
+}
+```
+
+Proposing to replace the fieldPath with urn in the ```SchemaField```
+
+```java
+record SchemaField {
+
+  /**
+   * Urn for the Dataset Field
+   */
+  urn: DatasetFieldUrn
+
```

Comment:
This is a breaking change and calls for a bigger discussion. 

IMO in the scope of this RFC, fieldPath is good enough. 
Because, datasetFiledUrn can be constructed using fieldPath + datasetUrn.